Data/Sequence/Internal.hs

```diff
@@ -4457,7 +4505,11 @@ unstableSortBy cmp (Seq xs) =
 -- | fromList2, given a list and its length, constructs a completely
 -- balanced Seq whose elements are that list using the replicateA
 -- generalization.
+#if __GLASGOW_HASKELL__ >= 800
+fromList2 :: HasCallStack => Int -> [a] -> Seq a
```

Comment:
This is an internal function and shouldn't get a call stack. It's used to implement the `IsList` instance. Arguably, the method it's used to implement should get a call stack, but that method isn't really intended to be called directly in user code anyway, and GHC will always call it correctly.